src/libs/actions/IOU.js

```diff
@@ -642,6 +642,7 @@ function requestMoney(report, amount, currency, created, merchant, payeeEmail, p
             createdIOUReportActionID,
             reportPreviewReportActionID: reportPreviewAction.reportActionID,
             receipt,
+            receiptState: lodashGet(receipt, 'state'),
```

Comment:
BE: This is redundant. We are sending duplicate data, receipt already have state, the backend should extract it from there instead of us having to send it again separately.

Reply:
Yea, I was trying to avoid that too, but our backend seems to strip away any custom properties from files. Since we were losing this data, I ended up having to send this as a separate param.